bridge/client/app/_services/api.service.ts

```diff
@@ -270,6 +271,16 @@ export class ApiService {
     return this.http.get<string[]>(url);
   }
 
+  public getStageNames(projectName: string): Observable<string[]> {
```

Comment:
I think it is not rocket science to implement this. I think we can remove it. If we need it because we move the component or any new feature needs it I think we can re-implement it.

Reply:
Also removed the parts in server/../data-service and api-service